### PR TITLE
Mqtt binding

### PR DESCRIPTION
**Pull request by hh6196**
## Main
Mqtt-commmunication-component(Mcc) separated from App.vue
### sub
Get parser's data to mq-re.vue(mcc)
Combine createConnection(), doSubscribe(), doPublish() into submit().

### CODE REVIEW

**Review by Citrus-Unshiu — COMMENTED** [review]
Very goooooood

File: oneM2M_Data_Model_Design_Tool/src/components/Modal.vue
Comment (LAMXkE) [inline oneM2M_Data_Model_Design_Tool/src/components/Modal.vue:1]: 이건 어쩌다가 생성된 것?

**Comment by JJ1iN** [comment]
you gosu

**Review by LAMXkE — APPROVED** [review]
Great

**Pull request merged**